src/ImmersedBoundaries/ImmersedBoundaries.jl

```diff
@@ -102,26 +102,27 @@ abstract type AbstractImmersedBoundary end
 ##### ImmersedBoundaryGrid
 #####
 
-struct ImmersedBoundaryGrid{FT, TX, TY, TZ, G, I, M, Arch} <: AbstractGrid{FT, TX, TY, TZ, Arch}
+struct ImmersedBoundaryGrid{FT, TX, TY, TZ, G, I, M, S, Arch} <: AbstractGrid{FT, TX, TY, TZ, Arch}
     architecture :: Arch
     underlying_grid :: G
     immersed_boundary :: I
     interior_active_cells :: M
-    
+    surface_active_cells :: S
```

Comment:
what surface is this, top, bottom, north, south?

Reply:
That is meant to be the top boundary. It comes from PR #3404. I have to correct it there.

Reply:
Actually, to be precise, it's the active map of reduced fields in the z-direction

Reply:
That is only relevant for the hydrostatic model right? It wouldn't apply to the non-hydrostatic model, or probably other models that are isotropic and 3D.